CHANGELOG.mdx

```diff
@@ -23,6 +23,9 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 - [#10840](https://github.com/rasahq/rasa/issues/10840): Fix issue with missing running event loop in `MainThread` when starting Rasa Open 
   Source for Rasa X with JWT secrets.
 
+### Miscellaneous internal changes
+- [#10712](https://github.com/RasaHQ/rasa/issues/10712): remove _warn_and_correct_transformer_size function as it's not needed anymore
+
```

Comment:
You shouldn't directly edit `CHANGELOG.mdx`. Here's one example of a changelog file:
![Screenshot from 2022-02-10 18-12-12](https://user-images.githubusercontent.com/45267439/153425054-df33375c-0082-4522-bfb6-391f80dbf6b6.png)
Please follow the instructions given [here](https://github.com/RasaHQ/rasa/tree/main/changelog#readme) and create a similar file.